test/data/manifests/fedora_37-aarch64-iot_commit_with_container-boot.json

```diff
@@ -13622,7 +13650,16 @@
       "Digest": "sha256:4d76a7480ce1861c95975945633dc9d03807ffb45c64b664ef22e673798d414b",
       "TLSVerify": null,
       "ImageID": "sha256:d4ee87dab8193afad523b1042b9d3f5ec887555a704e5aaec2876798ebb585a6",
-      "LocalName": "registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/fedora-minimal"
+      "LocalName": "registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/fedora-minimal",
+      "ListDigest": ""
```

Comment:
Although CI won't fail on this, I think that it would be good to regenerate these manifests, since `ListDigest` is now `omitempty`, thus it will disappear. But I'm OK with ignoring it as it has no impact (AFAICT).

Reply:
I was expecting it to go away as well, but it turns out that's now what we're serialising there, but instead the internal type: https://github.com/achilleas-k/osbuild-composer/blob/d694bd1cd5d15f06028e58f4412135aa4ea42003/cmd/gen-manifests/main.go#L267

We should probably have a serialisable type for this instead.